test/test_helper.rb

```diff
@@ -48,6 +48,8 @@ def initialize(cache_path)
       end
 
       def ensure_cached; end
+
+      def digest; '' end
     end
```

Comment:
[`RemoteSources::Common`](https://github.com/torba-rb/torba/blob/e333ad9edffb873a22b5e59735fd5bfcd0cd1f26/lib/torba/remote_sources/common.rb#L18-L22) requires subclasses to implement this in order for `build` to work properly.